xml/installation-ardana-manila.xml

```diff
@@ -70,38 +71,41 @@
 If &o_sharefs; has already been installed but is being reconfigured run the following for
 the changes to take effect:
 &prompt.ardana;ansible-playbook -i hosts/verb_hosts manila-stop.yml
-&prompt.ardana;ansible-playbook -i hosts/verb_hosts manila-start.yml</screen>
-   </step>
-   <step>
-    <para>
-     Workaround to fix manila-manage.log ownership problem &o_sharefs;
-    </para>
-<screen>on controller1:
-&prompt.ardana;cd /var/log/manila
-&prompt.ardana;sudo chown manila:manila manila-manage.log
-on &clm;:
-&prompt.ardana;ansible-playbook -i hosts/verb_hosts manila-stop.yml
 &prompt.ardana;ansible-playbook -i hosts/verb_hosts manila-start.yml</screen>
    </step>
    <step>
     <para>
      Verify the &o_sharefs; installation
     </para>
 <screen>&prompt.ardana;cd
-&prompt.ardana;. manila.osrc; . service.osrc; manila api-version; manila service-list</screen>
+&prompt.ardana;. manila.osrc
+&prompt.ardana;. service.osrc
+&prompt.ardana;manila api-version
+&prompt.ardana;manila service-list</screen>
     <para>
      The &o_sharefs; CLI can be run from &clm; or controller nodes.
     </para>
    </step>
   </procedure>
+  <note>
+   <para>
+    The <literal>manila-share</literal> service component is not started by the
+    <filename>manila-deploy.yml</filename> playbook under all conditions. This
```

Comment:
"... is not started when the <filename>manila-deploy.yml</filename> playbook is run under all conditions." (just helps clarify the context)